malasakit-django/cafe/settings.py

```diff
@@ -83,8 +84,10 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'ENGINE': 'django.db.backends.mysql',
+        'NAME': 'pcari',
+        'USER': 'root',
+        'PASSWORD': os.environ.get('mysql_pass'),
```

Comment:
Not sure how secure this is. However, if an attacker can read environment variables, it's probably already too late, and I don't have a good alternative in mind. We'll want to make sure to secure the production server.